// face.proto
syntax = "proto3";

service FaceRecognition {
  rpc RecognizeFace (ImageRequest) returns (RecognitionResponse);
}

message ImageRequest {
  bytes image_data = 1;
}

message RecognitionResponse {
  string name = 1;
  float confidence = 2;
  int32 x = 3;
  int32 y = 4;
  int32 w = 5;
  int32 h = 6;
}
